src/hooks/Video.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+import { VideoContext } from '../shared/VideoContext'
+
+const baseClass = 'video'
+
+export const Video = props => {
+  const videoClass =
+    props.size === 'large'
+      ? `${baseClass}__wrapper--large`
+      : `${baseClass}__wrapper`
+  const { setVideoRef, togglePlaying } = React.useContext(VideoContext)
+
+  return (
+    <video
+      className={videoClass}
+      onClick={togglePlaying}
+      ref={setVideoRef}
+      src={require('../shared/MacgruberTrailer.mp4')}
```

Comment:
This doesn't get re-required on every render, does it? Should this be required once at the top? Or is this intentional so rerenders do something special? Or maybe this require does only happen once? Maybe just a comment explaining this because off the cuff it seems surprising to see a `require` in a render

Reply:
Good question. I'm not 100% sure but I changed it to import the file instead of requiring it here.